igc_lib_test.py

```diff
@@ -3,7 +3,32 @@
 
 import igc_lib
 
+class TestNapretTaskParsing(unittest.TestCase):
 
+    def setUp(self):
+        test_file = 'testfiles/napret.lkt'
+        self.task = igc_lib.Task.create_from_lkt_file(test_file)
+
+    def testTaskHasStartTime(self):
+        self.assertAlmostEqual(self.task.start_time, 12*3600)
+
+    def testTaskHasEndTime(self):
+        self.assertAlmostEqual(self.task.end_time, 23*3600 + 59*60 + 59)
+
+    def testTaskHasTurnpoints(self):
+        self.assertEqual(len(self.task.turnpoints), 11)
+        self.assertEqual(self.task.turnpoints[0].kind, "start_enter")
+         
+    def testTaskHasTurnpointsWithRadius(self):
+        self.assertGreaterEqual(all(map(lambda turnpoint: turnpoint.radius, self.task.turnpoints)), 0.2)
+        self.assertLessEqual(all(map(lambda turnpoint: turnpoint.radius, self.task.turnpoints)), 4)
```

Comment:
These two lines work only by accident - the `all()` function returns a `bool`, in both cases `True`, which does compare correctly with the numbers: `True > 0.2`, `True < 4`.

What you need here is `max()`/`min()`:

```
     self.assertGreaterEqual(min(map(lambda turnpoint: turnpoint.radius, self.task.turnpoints)), 0.2)
      self.assertLessEqual(max(map(lambda turnpoint: turnpoint.radius, self.task.turnpoints)), 4)
```

Or an explicit list:
```
    self.assertEqual(len(self.task.turnpoints), 11)
    self.assertAlmostEqual(self.task.turnpoints[0].radius, XX)
    self.assertAlmostEqual(self.task.turnpoints[1].radius, XX)
    self.assertAlmostEqual(self.task.turnpoints[2].radius, XX)
    #...
```